src/validate.js

```diff
@@ -1659,6 +1659,18 @@ function F_VALIDATE(disabledHL) {
 		var reverseSpeed = segment.$revMaxSpeed;
 		var forwardSpeedUnverified = segment.$fwdMaxSpeedUnverified;
 		var reverseSpeedUnverified = segment.$fwdMaxSpeedUnverified;
+		var speedConversionFactor = 1;  // default to metric
+		var speedMultipleFactor = 10;   // default to limits being set in multiples of 10
+		// check for MPH countries
+		if ((country == "United Kingdom") ||
+			(country == "Jersey") ||
+			(country == "Guernsey") ||
+			(country == "United States"))
+			speedConversionFactor = 1.609;
```

Comment:
For km-to-miles conversion have a look at `OpenLayers.INCHES_PER_UNIT` and `W.map.units`